Makefile

```diff
@@ -105,14 +105,19 @@ bundler:
 .PHONY: gems
 gems: bundler
 	$(info:msg=Install/Update gems)
-	$(bundle_bin) install --jobs=4 --deployment --without development test
+	$(bundle_bin) config --delete with
```

Comment:
do we need it?
 $(bundle_bin) config --delete with

Reply:
Yes, because we get the following error: 
`a group cannot be in both `with` & `without` simultaneously`